package.json

```diff
@@ -201,7 +201,7 @@
     "yosay": "^1.2.1"
   },
   "devDependencies": {
-    "@ciscospark/eslint-config": "^1.0.0",
+    "@ciscospark/eslint-config": "^2.0.0",
```

Comment:
We'll need to change it on npm then because I believe @mmaines-cisco already published because we're using 2.0 in widgets.